src/single-spa-alpinejs.js

```diff
@@ -70,15 +98,43 @@ function mount(opts, props) {
       }
 
       domElement.innerHTML = template;
-      // TODO - create any global x-init functions that are needed
-      // TODO - pass single-spa props to the alpinejs application somehow. Maybe through domElement.__x.$data.singleSpaProps?
+      // TODO - create any global x-init,x-data functions that are needed
+      // setup opts x-data value
+      if (typeofXData !== "undefined") {
```

Comment:
A truthiness check here is better:

```suggestion
      if (opts.xData) {
```